resource-managers/kubernetes/integration-tests/pom.xml

```diff
@@ -26,8 +26,6 @@
 
   <artifactId>spark-kubernetes-integration-tests_2.12</artifactId>
   <properties>
-    <download-maven-plugin.version>1.3.0</download-maven-plugin.version>
-    <extraScalaTestArgs></extraScalaTestArgs>
```

Comment:
I think at some point we added things like this explicitly to override test args to be empty, because they were non-empty in the parent or something. I don't know if this is still the case though.

Reply:
According to my inspection, the `extraScalaTestArgs` parameter no longer exists in the project.
<img width="849" alt="image" src="https://user-images.githubusercontent.com/15246973/226231904-0af68136-3c2f-45c4-9f6f-9d0f50dcbe0d.png">

Reply:
OK looks good to me.

Reply:
So my guess is that `extraScalaTestArgs` can be helpful for local build/test for developers to add some custom java options, agents, etc. It just defaults to empty, not useless.